index.html

```diff
@@ -35,7 +35,119 @@ <h1>I'm David, Glad to see you</h1>
+            </ul>
+            <div>
+                <button type="button">See Project</button>
+            </div>
+            
+        </div>
+        <div class="card">
+            <img class="card-image" src="./img/portfolio3.jpg" alt="webpage image">
+            <h2>Tonic</h2>
+            <div class="tags">
+                <p>CANOPY</p>
+                <img class="bullet" src="./img/Counter.png" alt="bullet">
+                <span>Back End Dev</span>
+                <img class="bullet" src="./img/Counter.png" alt="bullet">
+                <span>2015</span>
+            </div>
+            <p class="description">
+                A daily selection of privately personalized reads. No accounts or 
+                sign-ups required.
+            </p>
+            <ul class="technologies">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+            </ul>
+            <div>
+                <button type="button">See Project</button>
+            </div>
+        </div>
+    </section>
+    <section class="about">
+        <div class="about-content">
+            <div class="container">
+                <h2>About myself</h2>
+                <p>Hello I'm a software developer! I can help you build a product,  
+                feature or website. Look through some of my work and experience!
+                If you like what you see and have a project you need coded, don't hesitate 
+                to contact me. 
+                </p>
+                <span>LET'S CONNECT</span>
+                <ul class="social-media">
+                    <li><a href="https://twitter.com/DavidEspinoG"><img src="./img/twitter.png" alt="twitter icon"></a></li>
+                    <li><a href="https://www.linkedin.com/in/davidespinog/"><img src="./img/linkedin.png" alt="linkedin icon"></a></li>
+                    <li><a href="#"><img src="./img/M.png" alt="M icon"></a></li>
+                    <li><a href="https://github.com/DavidEspinoG"><img src="./img/git.png" alt="github icon"></a></li>
+                    <li><a href="#"><img src="./img/hand.png" alt="hand icon"></a></li>
+                </ul>
+                <a><button>Get my resume</button></a>  
+
+                <div class="languages">
+                    <h3>Languages</h3>
+                    <i class="fa-solid fa-chevron-down fa-xl"></i> 
+                </div>
+                <div class="language">
+                    <img class="language-img" src="./img/js.png" alt="JavaScript image">
+                    <p>JavaScript</p>
+                </div>
+                <div class="language">
+                    <img class="language-img" src="./img/html.jpg" alt="JavaScript image">
+                    <p>HTML</p>
+                </div>
+                <div class="language">
+                    <img class="language-img" src="./img/css.jpg" alt="JavaScript image">
+                    <p>CSS</p>
+                </div>
```

Comment:
- [ ] Great job applying the previous reviewer's suggestions! 💯 However, there's still one that's not met and is using a list for these elements. Please consider refactoring this piece of code, to be a list, you could use the `<ul>` tag to achieve this. This will provide a better semantic value to your page since it is a list of languages. 🤓 🛠️ 

```suggestion
<ul>
    <li class="language">
        <img class="language-img" src="./img/js.png" alt="JavaScript image">
        <p>JavaScript</p>
    </li>
    <li class="language">
        <img class="language-img" src="./img/html.jpg" alt="JavaScript image">
        <p>HTML</p>
    </li>
    <li class="language">
        <img class="language-img" src="./img/css.jpg" alt="JavaScript image">
        <p>CSS</p>
    </li>
</ul>
```